db/migrate/20230613201911_create_posts.rb

```diff
@@ -0,0 +1,21 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :AuthorId
+      t.string :integer
+      t.string :Title
+      t.string :string
```

Comment:
- [ ] From the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) there is no column in the `Post` table as `string`. Kindly delete this column and ensure your migration follows the given ERD.